mlflow/lightgbm.py

```diff
@@ -281,7 +281,7 @@ def predict(self, dataframe):
 
 
 @experimental
-def autolog(log_input_examples=False, log_model_signatures=True):
+def autolog(log_input_examples=False, log_model_signatures=True, log_models=True):
```

Comment:
That's unfortunately a backwards-incompatible change at this point since the API has been released with a particular argument ordering for log_input_examples / log_model_signatures.